Gaijinizer patch/Map054.txt

```diff
@@ -980,75 +980,80 @@ disaster Ne, \\N[1].#MTLed
 だいぶ女悪魔の扱いに慣れてきたんじゃない？、
 女悪魔キラーの\\N[1]サン♪
 > CONTEXT: Map054/events/1/pages/5/21/Dialogue/hyakki
-Even though it is a lower-grade monster, it is
-important not to move to the temptation of that
-Te. Did you get used to the treatment of demon
-for a while? , Woman demon killer \\N[1] San♪#MTLed
+Even if they're low-rank youkai, that doesn't
+mean their temptation attacks won't affect you.
+Haven't you gotten used to dealing with female
+demons? Lady Demon Killer \\N[1]-San♪#Strate
 > END STRING
 
 > BEGIN STRING
 んで、そんなレディ・キラーなジゴロさんに忠告なんだけど…
 
 ちょっと今度マイルーム行くときは注意した方がいいかも…
 > CONTEXT: Map054/events/1/pages/5/29/Dialogue/hyakki
-Well, I'm advising such a lady killer Gigolo ...
-I should be careful when I go to my room for a
-moment ...#MTLed
+So, a little advice for Ladykiller Gigolo-san...
+Perhaps you should be careful next time you
+go to My Room...#Strate
 > END STRING
 
 > BEGIN STRING
 え、どういうこと…？
 > CONTEXT: Map054/events/1/pages/5/34/Dialogue/kazuya
-Eh, what kind of thing ...?#MTLed
+Eh, what do you mean...?#Strate
 > END STRING
 
 > BEGIN STRING
 じ、実はね……
 
 \\}ルシィさん…帰ってなかったの…
 > CONTEXT: Map054/events/1/pages/5/36/Dialogue/hyakki
-Really...\\}Lucy...I did not return ...#MTLed
+T-the truth is...
+
+\\}Lucy-san...didn't go back...#Strate
 > END STRING
 
 > BEGIN STRING
 \\}そ、それで……
 
 あの…、君とあの子達との…その…『現場』を……
 > CONTEXT: Map054/events/1/pages/5/40/Dialogue/hyakki
-\\}Well then...That ..., with you and his children ...
-that..."the scene" ...#MTLed
+\\}S-so......
+That..., with you and those kids...
+that..."scene"...#StrateUnsure
 > END STRING
```

Comment:
Note

Reply:
No issue with this from a JP perspective. She's basically nervously saying that Iron Maiden didn't leave straight away and was watching the scene play out with Kazuya and the nekomata.

Reply:
I think this works a bit better...
```
\\}S-so......
T-that... "scene"... with you...
and those... kids...
```